apps/web/pages/event-types/index.tsx

```diff
@@ -175,22 +174,17 @@ export const EventTypeList = ({ group, groupIndex, readOnly, types }: EventTypeL
   }
 
   // inject selection data into url for correct router history
-  const openModal = (group: EventTypeGroup, type: EventType) => {
+  const openDuplicateModal = (eventType: EventType) => {
```

Comment:
Some times we are better off creating a single function for a specific use case that using one and if else everything